index.html

```diff
@@ -471,13 +471,16 @@ <h2 id="collections">Collection Functions (Arrays or Objects)</h2>
         through a transformation function (<b>iteratee</b>). The <tt>iteratee</tt>
         is passed three arguments: the <tt>value</tt>, then the <tt>index</tt>
         (or <tt>key</tt>) of the iteration, and finally a reference to the entire
-        <tt>list</tt>.
+        <tt>list</tt>. Selective <b>Underscore</b> functions marked as <tt>guarded</tt>
+        can be used as an <tt>iteratee</tt> to <b>_.map</b>.
```

Comment:
There is nothing in the guard that prevents it from working with any other method like `_.map` that triggers it. For example:
```js
_.filter([[0,1,2],[3,4,5]], _.first);
// => [[3,4,5]]
```